packages/parse5-html-rewriting-stream/docs/index.md

```diff
@@ -1,163 +1,5 @@
 # Class: RewritingStream
 
-Streaming [SAX](https://en.wikipedia.org/wiki/Simple_API_for_XML)-style HTML rewriter. A [transform stream](https://nodejs.org/api/stream.html#stream_class_stream_transform) (which means you can pipe _through_ it, see example).
```

Comment:
Why remove docs?

Reply:
I moved a lot of the docs to JSDoc, which will make them accessible from editors. I haven't figured out how to document events yet, but that's the only reason why the original docs are still around.

@43081j — Do you have a suggestion of how to tackle events in JSDoc?

Reply:
events are a little odd in jsdoc, we can specify something like a `@fires` tag but that should also mean we define the event's type somewhere with `@event`. not entirely sure what convention vscode will look for though.

Reply:
Not everyone uses TypeScript, and not everyone learns how projects work by reading the code.
I personally have an **really** hard time dealing with projects that don’t have docs. While the current docs are scattered, I’m :-1: on removing most/all docs.

Reply:
The previous docs were already generated from TypeScript types, using `typedoc-plugin-markdown`. I would set up a website that makes them consumable outside of TypeScript, similar to http://cheerio.js.org